jobs/ble_test.yaml

```diff
@@ -34,8 +34,7 @@ actions:
           - REPOSITORY=""
           - SHA=""
           - GITHUB_CREDS=""
-          - bash /opt/lava-worker/mbed-os-lava-scripts/scripts/bootstrap.sh
-          - STATE="failure"
-          - bash /opt/lava-worker/mbed-os-lava-scripts/scripts/ble_test.sh "https://github.com/${REPOSITORY}.git" "$SHA" && STATE="success"
-          - if [ "$STATE" = "failure" ]; then lava-test-raise "FAILED"; fi
+          - apt-get update && apt-get install -y jq p7zip-full
```

Comment:
Why not keep a bootstrap operation ?

Reply:
one less file to maintain, one less call to make from the job

Reply:
My problem with it is that every lava job file will have to have that line and be updated if requirements change. So that's many more files to maintain.

Reply:
sorry, do you mean the `apt-get update && apt-get install -y jq p7zip-full`? That is temporary until I fix the docker image. But there may be other things required in the future, so I can put the bootstrap back in but launched by script themselves. I only had the bootstrap invocation separate so that it could update the scripts before running. Since we start fresh now there's no need for that so we can keep it simler.